build-system/compile/build.conf.js

```diff
@@ -21,7 +21,7 @@ const localPlugin = name =>
 
 const defaultPlugins = [
   // TODO(alanorozco): Remove `replaceCallArguments` once serving infra is up.
-  [localPlugin('transform-log-methods'), {replaceCallArguments: false}],
+  [localPlugin('transform-log-methods'), {replaceCallArguments: true}],
```

Comment:
`// TODO(me): before sending for review`

Turn this off since I just want it for a Travis run :)